backend/auth/auth_utils.py

```diff
@@ -21,19 +17,4 @@ def get_authenticated_user_details(request_headers):
     user_object['client_principal_b64'] = raw_user_object.get('X-Ms-Client-Principal')
     user_object['aad_id_token'] = raw_user_object.get('X-Ms-Token-Aad-Id-Token')
 
-    return user_object
-
-def get_tenantid(client_principal_b64):
```

Comment:
is tenant id no longer needed?

Reply:
it is needed, but currently it is a limitation - see https://github.com/Azure/static-web-apps/issues/897
It is not MVP so I removed it for now :)